### DISCUSSION TITLE
Injecting new attributes into BaseUser scheme?

### DISCUSSION THREAD

**Discussion by hsul-git** [fastapi-users/fastapi-users#1132] [Q&A]
Hi, 

is it possible to alter the structure of the BaseUser? Like adding the extra cloumns that i have created to the User instance that is used for dependency checkup on endpoints?

schemas.py:
```
import uuid

from fastapi_users import schemas


class UserRead(schemas.BaseUser[uuid.UUID]):
    is_db_read: bool
    is_db_cypher: bool
    is_db_write: bool


class UserCreate(schemas.BaseUserCreate):
    is_db_read: bool = True
    is_db_cypher: bool = False
    is_db_write: bool = False


class UserUpdate(schemas.BaseUserUpdate):
    is_db_read: bool
    is_db_cypher: bool
    is_db_write: bool
```

Changed part of the standard sqlite.py:
```
class User(fastapi_users.db.SQLAlchemyBaseUserTableUUID, Base):
    is_db_read = Column("is_db_read", Boolean, nullable=False)
    is_db_cypher = Column("is_db_cypher", Boolean, nullable=False)
    is_db_write = Column("is_db_write", Boolean, nullable=False)
```

So the writing to the database is working, but how to inject this into the current_user?
Do i have to monkey patch the fastapi-users classes?

Thx for helping.

**Comment by frankie567** [comment]
`current_user` will return you a valid SQLAlchemy object of your class `User`. So you'll have access to your properties.
